packages/common/src/store/ui/share-modal/sagas.ts

```diff
@@ -36,10 +43,41 @@ function* handleRequestOpen(action: ShareModalRequestOpenAction) {
     }
     case 'collection': {
       const { collectionId, source } = action.payload
-      const collection = yield* select(getCollection(collectionId))
+      const audiusSdk = yield* getContext('audiusSdk')
```

Comment:
the share modal wasn't opening for playlists that aren't owned by the user because the data was not cached / present, this change addresses that by fetching the data if not available